projects/core/components/scrollbar/scrollbar.component.ts

```diff
@@ -18,6 +18,8 @@ export function scrollRefFactory({
     return browserScrollRef;
 }
 
+type Css = CSS;
```

Comment:
It will break build without extra type declaration, won't it ?

Reply:
fixed